.editorconfig

```diff
@@ -14,3 +14,6 @@ trim_trailing_whitespace = true
 [*.yml]
 indent_style = space
 indent_size = 2
+
+[*.twig]
+insert_final_newline = false
```

Comment:
This shouldnt be neccessary right?

Reply:
It is since there's `insert_final_newline = true` for `[*]` above. The new line at end of twig file causes extra new lines to be added in the generated files causing tests to fail.

Reply:
maybe we could remove or trim here rather than trying to enforce this. will be difficult, as not everyone has editorconfig support

Reply:
They their editor doesn't support editorconfig then new line won't be auto added at end of file anyway :)

Reply:
Extra ones will also not be removed, though.
But I guess fair enough :)

Reply:
I just saw somewhere else:

   {{ x -}}

Could the `-` added be a trick to avoid newlines being added?

Reply:
Yes using `-}}` works but that would require adding such tag at end of practically every file, I don't think that's acceptable.

Reply:
Well for the regular view templates I think that's how it should work. Embedded Ruby works the same way AFAIR.

And I'd say for consistency we should do it everywhere. Just replace %> with -%> and then remove the - at some places?